pubspec.yaml

```diff
@@ -29,12 +29,9 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^1.0.0
-  built_value_generator:
-    git: 
-      url: https://github.com/efortuna/built_value.dart.git/
-      path: built_value_generator
+  built_value_generator: any
 
-  test: ^1.5.1
+  test: 1.6.2
```

Comment:
This needs to be pinned like that? Just checking.

Reply:
It was the only way I could get it to build.  Maybe we could unpin when we have more time to update all of the deps?